mrmustard/physics/triples.py

```diff
@@ -90,7 +90,7 @@ def vacuum_state_Abc(n_modes: int) -> Union[Matrix, Vector, Scalar]:
     """
     A = _vacuum_A_matrix(n_modes)
     b = _vacuum_B_vector(n_modes)
-    c = 1.0 + 0j
+    c = math.astensor(1.0 + 0j)
```

Comment:
```suggestion
    c = math.astensor(1.0 + 0.0j)
```
not important but it looks consistent with line 794